bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -24,14 +24,12 @@ class Neuron(CMakePackage):
     url = "http://www.neuron.yale.edu/ftp/neuron/versions/v7.5/nrn-7.5.tar.gz"
     git = "https://github.com/neuronsimulator/nrn"
 
-    # Patch which reverts 81a7a39 for numerical compat
-    patch("revert_Import3d_numerical_format.master.patch", when="@7.8.1:")
```

Comment:
We could also consider leaving the patch in Spack and setting `9.0.a6` as the end of the range.

Reply:
I was thinking the same but I since Alex completely removed it I thought there was some reason to keeping this. Anyway, I reverted the deletion

Reply:
:hydra: